test/integration/test-fixtures/image-rekor/Dockerfile

```diff
@@ -0,0 +1,2 @@
+# An image containing the example sample-golang-prov binary from https://github.com/lumjjb/sample-golang-prov
+FROM docker.io/mdeicas/sample-golang-prov:latest
```

Comment:
do we have a nice way to generate this without needing to rely on an external 3rd party docker image?

Reply:
One alternative is to put the `sample-golang-prov` binary in test-fixtures and have ```FROM scratch
COPY sample-golang-prov /```. Does this sound better?

If this change is made, then the same should probably be done for the `image-rust-auditable` docker file.  
@wagoodman